examples/runtime-v2/demo-flow/concord.yml

```diff
@@ -0,0 +1,101 @@
+# how to enable v2
+configuration:
+  runtime: "concord-v2"
+  entryPoint: "aFlow"
+  dependencies:
+    - "mvn://org.codehaus.groovy:groovy-all:pom:2.5.2"
+
+# new directory structure
+# new SDK
+
+  arguments:
+    url: "https://concord.test.walmart.com"
```

Comment:
This will cause vulnerabilities like [this](https://github.com/walmartlabs/concord-website/pull/37)

Reply:
good point